package.json

```diff
@@ -54,6 +57,7 @@
     ]
   },
   "dependencies": {
+    "@babel/polyfill": "^7.2.5",
```

Comment:
I’d strongly prefer to avoid this; i don’t want core-is in my dev dep graph. Why is it needed?

Reply:
Good point. It was there before. Let me try removing it.